src/server.ts

```diff
@@ -16,6 +16,16 @@ router.get("/nodes/:id", (ctx) => {
   ctx.body = getNodeAndChildren(Number(ctx.params.id))
 })
 
+router.post("/nodes", (ctx) => {
+  const body = ctx.request.query
+  ctx.body = storeNewNode(body.name, body.type, Number(body.parent))
+})
+
+router.del("/nodes/:ids", (ctx) => {
+  const ids = ctx.params.ids.split("-").map(Number)
+  ctx.body = deleteNodes(ids)
```

Comment:
I would just return empty string and status to confirm that modifications have been made